### PR TITLE
Add saving methods to date and killer commands

### PR DESCRIPTION
**Pull request by Diet-Cola**

### CODE REVIEW

**Comment by Diet-Cola** [comment]
Fixes CivMC/Civ#154

**Comment by Diet-Cola** [comment]
Just need to test

File: paper/src/main/java/com/devotedmc/ExilePearl/core/CoreExilePearl.java [hunk 1]
```diff
@@ -6,21 +6,9 @@
 import com.devotedmc.ExilePearl.PearlType;
 import com.devotedmc.ExilePearl.broadcast.BroadcastListener;
 import com.devotedmc.ExilePearl.event.PearlMovedEvent;
-import com.devotedmc.ExilePearl.holder.BlockHolder;
-import com.devotedmc.ExilePearl.holder.EntityHolder;
-import com.devotedmc.ExilePearl.holder.HolderVerifyResult;
-import com.devotedmc.ExilePearl.holder.ItemHolder;
-import com.devotedmc.ExilePearl.holder.PearlHolder;
-import com.devotedmc.ExilePearl.holder.PlayerHolder;
+import com.devotedmc.ExilePearl.holder.*;
```
Comment (Protonull) [inline paper/src/main/java/com/devotedmc/ExilePearl/core/CoreExilePearl.java:9]: Star imports :nauseated_face:

File: paper/src/main/java/com/devotedmc/ExilePearl/core/ExilePearlCore.java [hunk 2]
```diff
@@ -36,21 +14,10 @@
 import com.google.common.base.Preconditions;
 import com.wimbli.WorldBorder.BorderData;
 import com.wimbli.WorldBorder.WorldBorder;
+import io.papermc.paper.plugin.configuration.PluginMeta;
```
Comment (Protonull) [inline paper/src/main/java/com/devotedmc/ExilePearl/core/ExilePearlCore.java]: Curious, this interface doesn't exist in 1.18 ([reference](https://jd.papermc.io/paper/1.18/io/papermc/paper/plugin/configuration/PluginMeta.html)), but it does in 1.19 ([reference](https://jd.papermc.io/paper/1.19/io/papermc/paper/plugin/configuration/PluginMeta.html)), yet this PR is against master which is explicitly 1.18.

Comment (Diet-Cola) [reply paper/src/main/java/com/devotedmc/ExilePearl/core/ExilePearlCore.java]: Hmm I was working on 1.19 before working on this, maybe didnt cleanCache gradle before hand

File: paper/src/main/java/com/devotedmc/ExilePearl/core/CoreExilePearl.java (hunk 1 above)
Comment (Diet-Cola) [reply paper/src/main/java/com/devotedmc/ExilePearl/core/CoreExilePearl.java:9]: Not sure why my IDE has done this 🤔

**Comment by Protonull** [comment]
Also, any chance of CivMC/Civ#157 being included in this :P

**Comment by Diet-Cola** [comment]
Tested okay on test 👍

**Review by AngrySoundTech — APPROVED** [review]

**Pull request merged**